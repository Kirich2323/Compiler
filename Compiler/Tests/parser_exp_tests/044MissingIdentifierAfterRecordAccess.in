i + a.j - b. + 2