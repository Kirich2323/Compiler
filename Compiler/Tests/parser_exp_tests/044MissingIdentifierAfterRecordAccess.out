(1 ; 14): Got unexpected token: "+". Expected: "identifier".